client/web/src/integration/utils.ts

```diff
@@ -77,6 +78,19 @@ export const setColorScheme = async (
     await page.waitForTimeout(500)
 }
 
+const toggleRedesign = async (page: Page, enabled: boolean): Promise<void> => {
+    await page.evaluate(
+        (className: string, storageKey: string, enabled: boolean) => {
+            document.documentElement.classList.toggle(className, enabled)
+            localStorage.setItem(storageKey, enabled ? 'true' : 'false')
+            window.dispatchEvent(new StorageEvent('storage', { key: storageKey, newValue: enabled ? 'true' : 'false' }))
```

Comment:
nit: we could use string conversion for the boolean value

```suggestion
            localStorage.setItem(storageKey, String(enabled))
            window.dispatchEvent(new StorageEvent('storage', { key: storageKey, newValue: String(enabled) }))
```